src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+project(System.Net.Security.Native)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_definitions(-DPIC=1)
+
+if(APPLE)
+    add_definitions(-D__APPLE_USE_RFC_3542)
```

Comment:
Are these definitions being used anywhere?  I don't see them being used in this PR.